Makefile

```diff
@@ -97,7 +97,8 @@ vet:
 
 .PHONY: install-tools
 install-tools:
-	go install golang.org/x/lint/golint
+	GO111MODULE=on go install github.com/google/addlicense
```

Comment:
Is this needed with Go 1.12 and newer?

Reply:
Yes, this is still needed for Go 1.12 since it still has the same default as 1.11: "auto". Per https://github.com/golang/go/issues/31857 it seems that it will still be needed in Go 1.13.